ntp-udp/src/raw_socket.rs

```diff
@@ -157,7 +167,10 @@ mod set_timestamping_options {
             }
         };
 
-        configure_timestamping_socket(udp_socket, method, options)?;
+        configure_timestamping_socket(udp_socket, SockOpt::Method(method), options)?;
+
+        #[cfg(target_os = "freebsd")]
+        configure_timestamping_socket(udp_socket, SockOpt::Clock, libc::SO_TS_REALTIME as u32)?;
```

Comment:
This results in two calls to configure_timestamping_socket on freebsd, is that desirable?

Reply:
Never mind, I see what this is for